src/twosection.jl

```diff
@@ -0,0 +1,85 @@
+"""
+    TwoSectionView{T<:Real} <: AbstractGraph{Int64}
+
+Create a 2-section view of a hypergraph `h`.
+Note this is a view - changes to the original hypergraph will be automatically reflected in the view.
+
+The 2-section view of a hypergraph is suitable for processing with the LightGraphs.jl package.
+Several LightGraphs methods are provided for the compability.
+
+"""
+struct TwoSectionView{T<:Real} <: AbstractGraph{Int}
+    h::Hypergraph{T}
+    function TwoSectionView{T}(h::Hypergraph{T}) where {T<:Real}
+        new(h)
+    end
+end
+
+LightGraphs.nv(t::TwoSectionView) = length(t.h.v2he)
+
+LightGraphs.vertices(t::TwoSectionView) = Base.OneTo(nv(t))
+
+LightGraphs.ne(t::TwoSectionView)::Int64 = 0.5*sum(length.(t.h.he2v) .* (length.(t.h.he2v) .- 1))
+
+"""
+    LightGraphs.all_neighbors(t::TwoSectionView, v::Integer)
+
+Returns N(v) (the vertex v is not included in N(v))
+"""
+function LightGraphs.all_neighbors(t::TwoSectionView, v::Integer)
+    neighbors = Set()
+    for he in keys(t.h.v2he[v])
+        union!(neighbors, keys(t.h.he2v[he]))
+    end
+    delete!(neighbors, v) #remove v from its neighborhood
+    convert(Array{Int64}, collect(neighbors)) #returns the corresponding array
+    #neighbors
+end
+
+function LightGraphs.has_edge(t::TwoSectionView, s, d)
+    he_s = keys(t.h.v2he[s])
+    he_d = keys(t.h.v2he[d])
+
+    !issetequal(intersect(he_s, he_d), Set())
+end
+
+LightGraphs.outneighbors(t::TwoSectionView, v::Integer) = LightGraphs.all_neighbors(t::TwoSectionView, v)
```

Comment:
can you please fix indentation here and below?

Reply:
We have found a Julia [style guide](https://github.com/johnmyleswhite/Style.jl), where:  Never place more than 80 characters on a line. Is this the issue? or what you mean?

Reply:
actually I have thought that there was a line brak, but it was only the editor wrapping the code.
So from indentation we switch to line length as you ask.

[Here](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md) you have official guidelines. The customary line length limit is 92 and I would propose to follow this recommendation (as then when doing code review in two pane mode nothing wraps).